metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -36,26 +37,23 @@
  * @since 0.x
  */
 @Slf4j
-public class MetacatEventBus {
+public class MetacatEventBus extends SimpleApplicationEventMulticaster {
```

Comment:
A concern I have with this approach is the tight coupling this introduces to this Spring class. It's one thing to change how the event bus is created (constructor) it's another to extend this class because now we're exposing a bunch of API's that are only in Spring and could change and we're not in control of them anymore.

Reply:
The reason is how event handling works in Spring. We need MetacatEventBus to either implement the interface or extend the basic implementation. I prefer the later because it has other implementations that I am not clear about. Hence I left it as it is and made it the default event multicaster.

Reply:
I understand that I'm just still in favor of somehow doing this outside of manipulating the EventBus in this manner and somehow having whatever is sent into the EventBus be more correct. That way we don't really have to touch the event bus core logic code all the time...